hugr-core/src/extension/prelude.rs

```diff
@@ -95,14 +95,14 @@ lazy_static! {
                 TypeName::new_inline("usize"),
                 vec![],
                 "usize".into(),
-                TypeDefBound::Explicit(crate::types::TypeBound::Copyable),
+                TypeDefBound::Explicit { bound: crate::types::TypeBound::Copyable },
```

Comment:
you might consider factory functions `copyable()` and `any`/`linear()` on TypeDefBound

Reply:
nice idea, will do